README.md

```diff
@@ -1,51 +1,24 @@
-DNS query functions
-===================
+## Overview
 
-This module contains query functions for DNS for use from Puppet.
+This module provides functions for querying dns records
 
-Usage
------
+## Setup
 
-All functions can take a optional second lambda argument that is called if the lookup returned no values. Then the result of the lambda will be used instead.
+```console
+$ sudo puppet module install puppet-dnsquery
+```
 
-### dns_lookup
+## Usage
 
-Does a DNS lookup and returns an array of addresses.
+Since `puppet/dnsquery` version 4, all functions are provided under the `dnsquery` namespace.
+Non namespaced versions will be removed in version 5
 
-### dns_a
+All functions are documented in [REFERENCE.md](REFERENCE.md)
```

Comment:
I always question these relative links. For example, do they render well on the forge? But we do that more often so I'm not too worried about it.

Reply:
thanks and ftr i checked they dont work on forge but im not sure what the best thing to do in that case is.  do we want the link to redirect to github, or the rendered reference page on forge?  something to think.  ill merge this as is though